hasura/metadata/databases/default/tables/public_epoches.yaml

```diff
@@ -16,6 +16,13 @@ object_relationships:
         remote_table:
           name: circles
           schema: public
+  - name: epoch_pgive_data
```

Comment:
can you change the relation name to just `pgive_data`? no need to repeat `epoch_` when you're already starting from an epoch